src/main/kotlin/no/nav/melosysskattehendelser/skatt/sigrun/SigrunRestConsumer.kt

```diff
@@ -1,35 +1,37 @@
 package no.nav.melosysskattehendelser.skatt.sigrun
 
-import mu.KotlinLogging
 import no.nav.melosysskattehendelser.skatt.Hendelse
 import no.nav.melosysskattehendelser.skatt.HendelseRequest
 import no.nav.melosysskattehendelser.skatt.SkatteHendelseConsumer
 import org.springframework.web.reactive.function.client.WebClient
 import org.springframework.web.reactive.function.client.bodyToMono
 import java.time.LocalDate
 
-class SigrunRestConsumer(private val webClient: WebClient) : SkatteHendelseConsumer {
-    private val log = KotlinLogging.logger { }
+class SigrunRestConsumer(
+    private val webClient: WebClient,
+) : SkatteHendelseConsumer {
 
     override fun hentHendelseListe(request: HendelseRequest): List<Hendelse> =
-        webClient.get().uri("/api/skatteoppgjoer/hendelser")
-            .header("x-sekvensnummer-fra", request.seksvensnummerFra.toString())
-            .header("x-antall", request.antall.toString())
-            .header("x-bruk-aktoerid", request.brukAktoerId.toString())
+        webClient.get()
+            .uri { uriBuilder ->
+                uriBuilder.path("/api/v1/pensjonsgivendeinntektforfolketrygden/hendelser")
+                    .queryParam("fraSekvensnummer", request.seksvensnummerFra)
+                    .queryParam("antall", request.antall)
+                    .build()
+            }
+            .header("bruk-aktoerid", request.brukAktoerId.toString())
             .retrieve()
-            .bodyToMono<List<Hendelse>>()
-            .block() ?: emptyList()
+            .bodyToMono<Map<String, List<Hendelse>>>()
+            .block()?.get("hendelser") ?: emptyList()
```

Comment:
Finnes det noe annet enn hendelser?

Reply:
Nei, og utrolig rart at de har 2 endepunkter som i utgangspunktet her helt like men her så legger de det inn i en map med hendlers som peker til listen som vi vil ha. 
Men jeg vurdere nå om jeg skal logge warning eller kaste exception om vi ikke finner det vi forventer her. Så ikke ting stopper opp uten at vi ser feil i loggen

Reply:
Feil hvis man ikke finner "hendelser"? Høres bra ut for meg